drivers/net/wireless/ath/ath6kl/sdio.c

```diff
@@ -1408,6 +1408,7 @@ static void ath6kl_sdio_remove(struct sdio_func *func)
 }
 
 static const struct sdio_device_id ath6kl_sdio_devices[] = {
+	{SDIO_DEVICE(SDIO_VENDOR_ID_ATHEROS, 0x201)}, /* DSi/3DS wifi chip ID? */
```

Comment:
Do we want to leave this out until we have the base SDHC work landed?

Reply:
Nope, that was just an added ID to get stuff tested, will remove